src/command.rs

```diff
@@ -132,14 +136,22 @@ pub async fn remove(package: PackageName) -> miette::Result<()> {
     //     tracing::warn!("failed to uninstall package {}", dependency.package);
     // }
 
-    PackageStore::uninstall(&dependency.package).await.ok(); // temporary due to broken test
+    PackageStore::current()
+        .into_diagnostic()?
```

Comment:
Why is this needed? `PackageStore::xyz` should all return a `miette::Result` which you can just use with `?` directly

Reply:
This code is not from me, it is from @asmello, he said this was a workaround for a broken unit test.

Reply:
Oh wait, I misread. 

No, PackageStore returns a fine-grained error, which in this case is an `std::io::Error`. You cannot use the `?` operator with `miette::Result`. You always need to convert any errors into miette errors work `into_diagnostic()`.